Helper/PaymentMethods.php

```diff
@@ -252,15 +252,32 @@ protected function fetchAlternativeMethods($country)
         return $paymentMethods;
     }
 
-    /**
-     * @return bool|int
-     */
-    protected function getCurrentPaymentAmount()
+    protected function getCurrentPaymentAmount(): float
     {
-        if (($grandTotal = $this->getQuote()->getGrandTotal()) > 0) {
+        $grandTotal = $this->getQuote()->getGrandTotal();
+
+        if (!\is_numeric($grandTotal)) {
+            throw new \Exception(
+                \sprintf(
+                    'Cannot retrieve a valid grand total from quote ID: `%s`. Expected a numeric value.',
+                    $this->getQuote()->getEntityId()
+                )
+            );
+        }
+
+        $grandTotal = (float) $grandTotal;
+
+        if ($grandTotal > 0.0) {
             return $grandTotal;
         }
-        return 10;
+
+        throw new \Exception(
+            \sprintf(
```

Comment:
I think there's no need to specify the global namespace for these functions?
```suggestion
            sprintf(
```